src/wallet/wallet_rpc_server.cpp

```diff
@@ -615,22 +624,47 @@ namespace tools
         return false;
       }
 
-      if (!req.do_not_relay)
-        m_wallet->commit_tx(ptx_vector);
-
-      // populate response with tx hash
       res.tx_hash = epee::string_tools::pod_to_hex(cryptonote::get_transaction_hash(ptx_vector.back().tx));
```

Comment:
For multisig, isn't this inappropriate since the tx is incomplete at this point?